spec/mailers/gsi_mailer_spec.rb

```diff
@@ -1,5 +1,5 @@
-require "spec_helper"
+require 'spec_helper'
 
 describe GsiMailer do
-  pending "add some examples to (or delete) #{__FILE__}"
+  pending 'add some examples to (or delete the file)'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.